Sanitize

das-g opened the pull request:
Please keep the repo clean :stuck_out_tongue:

das-g committed:
remove NULL characters

das-g committed:
remove .DS_Store file

sykaeh reviewed (changes requested):
What did you change here? For some reason GitHub thinks that static-website/html.md is a binary file. Could you resolve the conflict?

das-g committed:
Merge branch 'master' into sanitize

das-g commented:
> What did you change here?

What the commit messages say ;-)

- `static-website/html.md` contained some [`NULL` characters](https://en.wikipedia.org/wiki/Null_character) (The ones you'd get in Python with `'\x00'`) that definitely don't belong in an ordinary text file, let alone a MarkDown file. I removed them. (No idea how they got in there. No text editor software I know of would insert them.)
- I've deleted the OSX-specific `.DS_store` file that was probably checked in by accident and isn't needed in the repository.

> For some reason GitHub thinks that static-website/html.md is a binary file.

Yeah, I noticed that, too. Probably because the change "contains" characters (the removed ones) that don't ordinarily occur in non-binary files.

> Could you resolve the conflict?

Done: b4aad6c
